### PR TITLE
RavenDB-10278 Always so the request URL and the full exception for ea…

### PR DESCRIPTION
**Pull request by fitzchak**
…ch node

### CODE REVIEW

File: src/Raven.Client/Http/RequestExecutor.cs [hunk 1]
```diff
@@ -775,28 +775,10 @@ private void ThrowFailedToContactAllNodes<TResult>(RavenCommand<TResult> command
...
 
-            var firstException = command.FailedNodes.Values.First();
-            
-            // this means that all exception has the same type
-            if (command.FailedNodes.All(x=> ExceptionAreEqual(x.Value, firstException)))
-            {
-                throw new AllTopologyNodesDownException(message, _nodeSelector?.Topology, firstException);
-            }
-
             throw new AllTopologyNodesDownException(message, _nodeSelector?.Topology,
```
Comment (ayende) [inline src/Raven.Client/Http/RequestExecutor.cs]: Here, I want the message to also include:

[ URL Used -> Error message ], [ URL Used 2 -> Error message ], [ URL Used  3-> Error message ],

**Review by ayende — CHANGES_REQUESTED** [review]

File: src/Raven.Client/Http/RequestExecutor.cs (hunk 1 above)
Comment (fitzchak) [reply src/Raven.Client/Http/RequestExecutor.cs]: Done

File: src/Raven.Client/Http/RequestExecutor.cs [hunk 2]
```diff
@@ -762,17 +762,28 @@ private void ThrowFailedToContactAllNodes<TResult>(RavenCommand<TResult> command
...
-                message += $"I've tried to access the following nodes: {string.Join(",", _nodeSelector?.Topology.Nodes.Select(x => x.Url) ?? new string[0])}.";
+                message += "Topology:";
+                foreach (var node in nodes)
+                {
+                    command.FailedNodes.TryGetValue(node, out var exception);
+                    message += Environment.NewLine +
+                               $"[Url: {node.Url}, " +
+                               $"ClusterTag: {node.ClusterTag}, " +
+                               $"ServerRole: {node.ServerRole}, " +
+                               $"Exception: {exception.ToString() ?? "No exception"}]";
```
Comment (ayende) [inline src/Raven.Client/Http/RequestExecutor.cs:785]: I think that we need just the message here, the full details are in the inner exception anyway

**Review by ayende — CHANGES_REQUESTED** [review]

File: src/Raven.Client/Http/RequestExecutor.cs (hunk 2 above)
Comment (fitzchak) [reply src/Raven.Client/Http/RequestExecutor.cs:785]: Right. I see you already made that change.

**Pull request merged**